Traceback (most recent call last):
  File "main.py", line 1, in <module>
    a = int(input())
EOFError: EOF when reading a line
